### PR TITLE
DevKit updates for 1.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -51,8 +51,12 @@ matrix:
     - php: '7.1'
       env: SYMFONY=dev-master@dev
     - php: '7.1'
+      env: SONATA_ADMIN=3.*
+    - php: '7.1'
       env: SONATA_ADMIN=dev-master@dev
     - php: '7.1'
+      env: SONATA_BLOCK=3.*
```
Comment (dbu) [inline .travis.yml]: according to travis-ci, this is not installable

Comment (greg0ire) [inline .travis.yml]: Link for the lazy: https://travis-ci.org/sonata-project/SonataDoctrinePhpcrAdminBundle/jobs/262050124#L467

Comment (greg0ire) [inline .travis.yml]: Maybe that's indirectly because of this: https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/blob/1.x/composer.json#L22

Comment (greg0ire) [inline .travis.yml]: https://github.com/sonata-project/dev-kit/pull/262

**Comment by greg0ire** [comment]
Should become green tonight

**Pull request merged**